app/src/main/java/org/mozilla/focus/activity/SettingsActivity.java

```diff
@@ -15,7 +15,7 @@
 import org.mozilla.focus.locale.LocaleAwareAppCompatActivity;
 import org.mozilla.focus.settings.SettingsFragment;
 
-public class SettingsActivity extends LocaleAwareAppCompatActivity implements SettingsFragment.TitleUpdater {
+public class SettingsActivity extends LocaleAwareAppCompatActivity implements SettingsFragment.ActionBarUpdater {
     public static final int ACTIVITY_RESULT_LOCALE_CHANGED = 1;
 
     @Override
```

Comment:
\[PMD\] INFO: The class 'SettingsActivity' has a Cyclomatic Complexity of 2 \(Highest = 5\)\.